recording_server/src/utils/takeScreenshot.ts

```
@@ -1,35 +1,35 @@
 import { Page } from '@playwright/test'
-import { promises as fs } from 'fs'
 import * as path from 'path'
-import sharp from 'sharp'
 import { PathManager } from '../utils/PathManager'
 import { s3cp } from './S3Uploader'
+import sharp from 'sharp'
+import { promises as fs } from 'fs'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)